IoTuring/Configurator/Configurator.py

```diff
@@ -1,13 +1,19 @@
 import inspect  # To get this folder path and reach the configurations file
-import os  # Configurations file path manipulation
-import json
+import os
 from IoTuring.Logger.LogObject import LogObject
+from IoTuring.Logger.Logger import Colors
 
 from IoTuring.ClassManager.EntityClassManager import EntityClassManager
 from IoTuring.ClassManager.WarehouseClassManager import WarehouseClassManager
 
 from IoTuring.Configurator.MenuPreset import MenuPreset
 
+import IoTuring.Configurator.ConfiguratorIO as ConfiguratorIO
+
+# TODO Find new location for this message
+HELP_MESSAGE = "\nYou can find the configuration file in the following path: \n\tmacOS\t\t~/Library/Application Support/IoTuring/configurations.json \n\tLinux\t\t~/.config/IoTuring/configurations.json \n\tWindows\t\t%APPDATA%/IoTuring/configurations.json\n\tFallback\t[ioturing_install_path]/Configurator/configurations.json \n\nYou can also set you preferred path that will contain the configuration file by setting the environment variable " + ConfiguratorIO.CONFIG_PATH_ENV_VAR + " (file will be stored in $" + ConfiguratorIO.CONFIG_PATH_ENV_VAR + "/configurations.json).\n"
```

Comment:
Related, we should start to use some linting standards. I know that my code is also chaotic, but the mixture of double and single quotes are starting to annoy me. 

[autopep8](https://pypi.org/project/autopep8/) is very well built into vscode, ctrl+shift+I is in my muscle memory before saving. But I don't use it on this project, because I don't want to reformat your files, messing up the git line history. https://stackoverflow.com/questions/29973357/how-do-you-format-code-in-visual-studio-code-vscode